src/libs/actions/ReimbursementAccount/navigation.js

```diff
@@ -17,9 +17,10 @@ function goToWithdrawalAccountSetupStep(stepID, newAchData) {
  * Navigate to the correct bank account route based on the bank account state and type
  *
  * @param {String} policyId
+ * @param {string} [backTo=''] - An optional return path. If provided, it will be URL-encoded and appended to the resulting URL.
```

Comment:
It's good we have the `backTo` property described here, but now this JSDoc is inconsistent because the `policyId` has no description - could we add it?